xcb.c

```diff
@@ -233,8 +235,14 @@ void grab_pointer_and_keyboard(xcb_connection_t *conn, xcb_screen_t *screen, xcb
         }
     }
 
-    if (tries <= 0)
+    /* After trying for 10000 times, i3lock will display an error message
+     * for 2 sec prior to terminate. */
+    if (tries <= 0) {
+        pam_state = STATE_PAM_FAIL;
+        redraw_screen();
+        usleep(2000000);
```

Comment:
Got you for `sleep`, it makes sense. For the sleep time, 2 sec was just a proposition, not to be displayed in a flash. But 1 sec is enough anyway. :-)